frontend/app/controllers/Giraffe.scala

```diff
@@ -41,10 +42,12 @@ object Giraffe extends Controller {
       navigation = Seq.empty,
       customSignInUrl = Some((Config.idWebAppUrl / "signin") ? ("skipConfirmation" -> "true"))
     )
-    Ok(views.html.giraffe.contribute(pageInfo,maxAmount,CountryGroup.UK))
+    Ok(views.html.giraffe.contribute(pageInfo,maxAmount,CountryGroup.UK,isUAT))
   }
 
-  def contributeUSA = CachedAction { implicit request =>
```

Comment:
I think we're at the point where we need to consider the viability of three separate, identical controllers. (But not as part of this PR)